contracts/FlowStakingCollection.cdc

```diff
@@ -388,37 +408,37 @@ pub contract FlowStakingCollection {
             }
 
             // If staking as a delegator, use the delegate functionality
-            if let _delegatorID = delegatorID {       
+            if let delegatorID = delegatorID {       
                 // If the delegator is stored in the collection, borrow it         
-                if let delegator = self.borrowDelegator(nodeID, _delegatorID) {
+                if let delegator = self.borrowDelegator(nodeID: nodeID, delegatorID: delegatorID) {
                     delegator.delegateNewTokens(from: <- self.getTokens(amount: amount))
                 } else {
                     // Get any needed unlocked tokens, and deposit them to the locked vault.
                     let lockedBalance = self.lockedVault!.borrow()!.balance - FlowStorageFees.minimumStorageReservation
                     if (amount > lockedBalance) {
-                        self.tokenHolder!.borrow()!.deposit(from: <- self.unlockedVault.borrow()!.withdraw(amount: amount - lockedBalance))
+                        let numUnlockedTokensToUse = amount - lockedBalance
+                        self.tokenHolder!.borrow()!.deposit(from: <- self.unlockedVault.borrow()!.withdraw(amount: numUnlockedTokensToUse))
```

Comment:
Maybe also refactor those borrows out to the top (`self.tokenHolder!.borrow()!` and `self.unlockedVault.borrow()!`), as they're done in both if-branches. It would make the code here more readable